package.json

```diff
@@ -112,9 +113,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.2.2",
-    "@babel/plugin-proposal-class-properties": "^7.1.0",
-    "@neutrinojs/mocha": "9.0.0-rc.0",
-    "@neutrinojs/node": "9.0.0-rc.0",
+    "@babel/plugin-proposal-class-properties": "^7.4.4",
```

Comment:
Why do these two stick around?  I tried removing them and `ui` still built properly..

Reply:
`@babel/plugin-proposal-class-properties` is being used by https://github.com/taskcluster/taskcluster/blob/6ae3183757756fd9028538436d21669636c9c82c/clients/client-web/.neutrinorc.js#L12 The latter has `@babel/core` specified as a peer dependency in https://github.com/babel/babel/blob/16e17c560b4d6c818b5ededcb0bbe6d6bf671f56/packages/babel-plugin-proposal-class-properties/package.json#L19.

Reply:
Indeed, I forgot about client-web.  It looks like that could be moved to `clients/client-web/package.json`, but I can do that in another PR.